pkgs/tools/security/wordlists/dirbuster.nix

```diff
@@ -0,0 +1,29 @@
+{ fetchurl
+, stdenv
+, unzip
+}:
+
+stdenv.mkDerivation  rec {
+  pname = "DirBuster";
+  version = "1.0-RC1";
+
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/dirbuster/files/DirBuster%20%28jar%20%2B%20lists%29/${version}/${pname}-${version}.zip";
```

Comment:
```suggestion
    url = "mirror://sourceforge/projects/dirbuster/files/DirBuster%20%28jar%20%2B%20lists%29/${version}/${pname}-${version}.zip";
```

Reply:
This does not work: it fetches an HTML document, and not the ZIP file I am expecting.
How can I browse the content of `sourceforge`'s mirror to locate the resource I am looking for?

Reply:
I've updated url, try it now.

Reply:
`mirror://sourceforge/` is:
https://github.com/NixOS/nixpkgs/blob/dc0d766f2dc06a01e2e3b35d72122cc22b631a35/pkgs/build-support/fetchurl/mirrors.nix#L19-L27